data/projects/integrate-pyflakes-AST.md

```diff
@@ -0,0 +1,16 @@
+Integrate the [`pyflakes`](https://pypi.python.org/pypi/pyflakes)-enhanced [AST](https://en.wikipedia.org/wiki/Abstract_syntax_tree) into coala so that bears can access it, and create two bears that wrap the flake8 AST plugins [`flake8-future-import`](https://github.com/xZise/flake8-future-import) and [`flake8-string-format`](https://github.com/xZise/flake8-string-format) using the pyflakes-enhanced AST.
```

Comment:
We can add that these flake8 plugins need to be installed into a dynamically created venv.
https://gitlab.com/coala/package_manager/issues/25